packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -44,13 +45,15 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
     private _buildTag?: string
     private _buildIdentifier?: string
     private _accessibilityAutomation?: boolean
+    public static _testOpsBuildStopped?: boolean
```

Comment:
Can keep as an instance variable

Reply:
This is getting used in utils. We have to created a new instance for `BrowserstackLauncherService` class if we implement an instance variable